docs/install/authentication-mechanisms-gcp.md

```diff
@@ -237,26 +239,26 @@ for automated scripts.
   `CloudAuditLogsSource`, `CloudPubSubSource`, etc.) can cause the Kubernetes
   Service Account `default-cre-dataplane` to be created. If they can also create
   Pods in that namespace, then they can make a Pod that uses the Google Service
-  Account `cre-dataplane` credentials.
+  Account `events-sources-gsa` credentials.
 
 ### Option 2. Export Service Account Keys And Store Them as Kubernetes Secrets
 
 1. Download a new JSON private key for the Google Cloud Service Account
-   `cre-dataplane` created in
+   `events-sources-gsa` created in
    [Installing a Service Account for the Data Plane](../install/dataplane-service-account.md).
    **Be sure not to check this key into source control!**
 
    ```shell
-   gcloud iam service-accounts keys create cre-dataplane.json \
-   --iam-account=cre-dataplane@$PROJECT_ID.iam.gserviceaccount.com
+   gcloud iam service-accounts keys create events-sources-key.json \
+     --iam-account=events-sources-gsa@$PROJECT_ID.iam.gserviceaccount.com
    ```
 
 1. Create a secret on the Kubernetes cluster with the downloaded key. Remember
    to create the secret in the namespace your resources will reside. The example
    below does so in the `default` namespace.
 
    ```shell
-   kubectl --namespace default create secret generic google-cloud-key --from-file=key.json=cre-dataplane.json
```

Comment:
@grac3gao should I replace `google-cloud-key` here with `google-cloud-sources-key`?

Reply:
Probably not. By default, the source is consuming `google-cloud-key` (webhook will add this to source).

Reply:
Thank you!